OrchardCore.Commerce.Tests/MoneyServiceTests.cs

```diff
@@ -45,16 +45,6 @@ public void NotFoundDefaultCurrencyFallsBackToDollar()
                     })).DefaultCurrency.IsoCode);
         }
 
-        [Fact]
-        public void CanEnumerateCurrenciesAcrossProviders()
```

Comment:
Why remove this test?

Reply:
re-implemented but just ensures all 114 items present.

Reply:
re implemented

Reply:
Does this still test that if there's another currency provider, adding custom currencies, we'd enumerate across all of them. Example: I add an Ethereum currency, or a "Contoso Reward Points" currency.